dspace-api/pom.xml

```diff
@@ -537,6 +618,35 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-core</artifactId>
+            <version>${solr.client.version}</version>
+        </dependency>
+        <!-- Reminder: Keep icu4j (in Parent POM) synced with version used by lucene-analyzers-icu below,
+             otherwise ICUFoldingFilterFactory may throw errors in tests.  -->
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-analyzers-icu</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-analyzers-smartcn</artifactId>
+            <version>${solr.client.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-analyzers-stempel</artifactId>
+            <version>${solr.client.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.xmlbeans</groupId>
+            <artifactId>xmlbeans</artifactId>
+            <version>2.6.0</version>
```

Comment:
This new dependency is also a bit odd...we don't have this one specified on `main` branch.  What this a dependency conversion fix?

Reply:
PoiWordFilter imports from xmlbeans.  `mvn dependency:analyze` showed xmlbeans as "used undeclared".  Even though it seemed to work in practice without the declaration, I tend to tidy these up whenever I become aware of them.  If I don't know why it worked, then I don't know why it might stop working.

Reply:
Yes, I see that now after looking closer at that PoiWordFilter. This is correct then...it just is slightly unrelated to this PR, but that's OK to clean this up :)